web3/utils/module_testing/eth_module.py

```diff
@@ -229,7 +230,19 @@ def test_eth_sendTransaction(self, web3, unlocked_account):
             ),
         ]
     )
-    def test_eth_sendRawTransaction(self, web3, raw_transaction, expected_hash):
+    def test_eth_sendRawTransaction(self,
+                                    web3,
+                                    unlocked_account,
+                                    raw_transaction,
+                                    expected_hash):
+        txn_params = {
+            'from': unlocked_account,
+            'to': "0x39EEed73fb1D3855E90Cbd42f348b3D7b340aAA6",
+            'value': (21000 * 100000000000 + 1),  # tx.gas * tx.gasprice + tx.value
+            'gas': 21000,
+            'gas_price': web3.eth.gasPrice,
+        }
+        txn_hash = web3.eth.sendTransaction(txn_params)
```

Comment:
This shouldn't be required.  Part of the necessary test setup for these tests *should* be pre-loading the raw transaction sender with the appropriate ether.

Reply:
Cool! Thank you for the solution